kaztron/cog/dice.py

```diff
@@ -84,6 +86,25 @@ def __init__(self, bot):
         await self.bot.say("[{}]\n**Sum:** {:d}".format(' '.join(rolls_str), total))
         logger.info("Rolled FATE dice: {!r} (sum={})".format(rolls_str, total))
 
+    # strip_list = map(str.strip, lines)
+    @commands.command(pass_context=True, ignore_extra=False, no_pm=False)
+    @in_channels(ch_allowed_list)
+    async def choose(self, ctx, *, choices: str):
+        "Need some help making a decision? Just give the .choose command two or more options(separated by ,) and it will choose for you!"
+        choices = list(map(str.strip, choices.split(",")))
+        if "" in choices:
+            logger.warning("choose(): argument empty")
+            await self.bot.say("I cannot decide if there's an empty choice.")
+        elif len(choices) < 2:
+            logger.warning("choose(): arguments out of range")
+            await self.bot.say("I need something to choose from.")
+        elif len(choices) > self.max_choices:
+            logger.warning("choose(): arguments out of range")
+            await self.bot.say("That is too much to choose from for me.")
```

Comment:
UX - Error msg not really clear - would suggest `"I don't know, that's too much to choose from! I can't handle more than {:d} choices!".format(self.max_choices)`